lib/ansible/modules/cloud/vmware/vmware_host_tcpip_stacks.py

```diff
@@ -0,0 +1,466 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2018, Christian Kotte <christian.kotte@gmx.de>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = r'''
+---
+module: vmware_host_tcpip_stacks
+short_description: Manage TCP/IP stack configuration of an ESXI host system
+description:
+- This module can be used to configure system TCP/IP stacks on an ESXi host system.
+- The DNS configuration of the default TCP/IP stack needs to be done with the vmware_host_dns module.
+version_added: 2.8
+author:
+- Christian Kotte (@ckotte)
+notes:
+- Tested on vSphere 6.5
+- You cannot remove the default gateway from the (vMotion) TCP/IP stack if you have a VMkernel adapter configured for this TCP/IP stack
```

Comment:
```suggestion
- You cannot remove the default gateway from the (vMotion) TCP/IP stack, if you have a VMkernel adapter configured for this TCP/IP stack.
```

Reply:
This isn't proper english. You have to use comma if you start with `if`, but not if `if` is at the "end" of the sentence.

```
If I go to London, I will visit the Tower.
I will visit the Tower if I go to London.
```